java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgPane.java

```diff
@@ -1627,7 +1639,14 @@ protected void addListener(java.beans.PropertyChangeListener qc) {
     }
 
     /**
-     * Create a new grid group from the JDOM group Element
+     * Create a new grid group from the JDOM group Element.
+     *
+     * @param element     element containing group contents
+     * @param c           the panel to create the grid in
+     * @param g           the layout manager for the panel
+     * @param globs       ???
```

Comment:
@dheap What are the GridGlobals and how are they used? What is the showStdName parameter?